.changes/v3.6.0/797-improvements.md

```diff
@@ -0,0 +1,4 @@
+* `resource/vcd_org` add support for `metadata` so this field can be set when creating/updating organizations. [GH-797]
+* `datasource/vcd_org` add support for `metadata` so this field can be retrieved when reading from organizations. [GH-797]
+* `resource/vcd_disk` add support for `metadata` so this field can be set when creating/updating independent disks. [GH-797]
```

Comment:
vcd_independent_disk or vcd_disk ? Also vcd_independent_disk changes are duplications of #789

Reply:
Fixed changelog to `vcd_independent_disk`. About duplications, you can go ahead with #789 and I'll pick your changes, np.